test/module/index.js

```diff
@@ -98,44 +98,40 @@ describe('Module Tests', function() {
   it('should throw an error when the first parameter is not defined', function() {
     const lighthouse = require('../..');
     return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is an empty string', function() {
     const lighthouse = require('../..');
-    return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse('')
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is an empty string');
+      }, err => {
+        assert.ok(err);
```

Comment:
If the module itself throws an error, everything is OK. If you throw an error in the test from the then callback, it's not OK.

In this case, it'll always not be ok right?

I think this is was I used callbacks in the original PR, avoid this confusion / unclarity.